### PR TITLE
brave_stats: Change ads_enabled pref_service

### PR DESCRIPTION
**Pull request by keur**
Resolves https://github.com/brave/brave-browser/issues/13765

Resolves 

Uplift of https://github.com/brave/brave-core/pull/7714

cc @rebron

### CODE REVIEW

**Comment by kjozwiak** [comment]
`Win` was having issues with `gn-audit-and-test-unit` due to `npm run gn_check -- Release` failing as per:

```
Cannot run program "nohup" (in directory "C:\ease"): CreateProcess error=2, The system cannot find the file specified
```

However, @mihaiplesa addressed the issue via https://github.com/brave/devops/pull/4339 and there's no need to restart/block uplift.

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.20.x` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on Nightly as per https://github.com/brave/brave-core/pull/7714#issuecomment-771088011.

**Comment by keur** [comment]
thanks @kjozwiak

**Pull request merged**
